youtube_dl/extractor/youtube.py

```diff
@@ -1809,6 +1813,7 @@ def feed_entry(name):
             'categories': [category] if category else None,
             'tags': keywords,
             'is_live': is_live,
+            'broadcast_start_timestamp': broadcast_start_timestamp,
```

Comment:
Rather than introducing a new metadata parameter, why shouldn't this just be the `timestamp` of the item, when it's a live show?
>    timestamp:      UNIX timestamp of the moment the video became available